bindings/node.js/src/kzg.cxx

```diff
@@ -376,173 +388,201 @@ Napi::Value VerifyKzgProof(const Napi::CallbackInfo& info) {
  */
-Napi::Value VerifyBlobKzgProof(const Napi::CallbackInfo& info) {
-  Napi::Env env = info.Env();
-  Blob *blob_bytes = get_blob(env, info[0]);
-  if (blob_bytes == nullptr) {
-    return env.Undefined();
-  }
-  Bytes48 *commitment_bytes = get_bytes48(env, info[1], "commitmentBytes");
-  if (commitment_bytes == nullptr) {
-    return env.Undefined();
-  }
-  Bytes48 *proof_bytes = get_bytes48(env, info[2], "proofBytes");
-  if (proof_bytes == nullptr) {
-    return env.Undefined();
-  }
-  KZGSettings *kzg_settings = get_kzg_settings(env, info);
-  if (kzg_settings == nullptr) {
-    return env.Undefined();
-  }
-
-  bool out;
-  C_KZG_RET ret = verify_blob_kzg_proof(
-    &out,
-    blob_bytes,
-    commitment_bytes,
-    proof_bytes,
-    kzg_settings);
-
-  if (ret != C_KZG_OK) {
-    std::ostringstream msg;
-    msg << "Error in verifyBlobKzgProof: " << from_c_kzg_ret(ret) ;
-    Napi::Error::New(env, msg.str()).ThrowAsJavaScriptException();
-    return env.Undefined();
-  }
+Napi::Value VerifyBlobKzgProof(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+    Blob *blob_bytes = get_blob(env, info[0]);
+    if (blob_bytes == nullptr) {
+        return env.Null();
+    }
+    Bytes48 *commitment_bytes = get_bytes48(env, info[1], "commitmentBytes");
+    if (commitment_bytes == nullptr) {
+        return env.Null();
+    }
+    Bytes48 *proof_bytes = get_bytes48(env, info[2], "proofBytes");
+    if (proof_bytes == nullptr) {
+        return env.Null();
+    }
+    KZGSettings *kzg_settings = get_kzg_settings(env, info);
+    if (kzg_settings == nullptr) {
+        return env.Null();
+    }
+
+    bool out;
+    C_KZG_RET ret = verify_blob_kzg_proof(
+        &out, blob_bytes, commitment_bytes, proof_bytes, kzg_settings
+    );
 
-  return Napi::Boolean::New(env, out);
+    if (ret != C_KZG_OK) {
+        std::ostringstream msg;
+        msg << "Error in verifyBlobKzgProof: " << from_c_kzg_ret(ret);
+        Napi::Error::New(env, msg.str()).ThrowAsJavaScriptException();
+        return env.Undefined();
+    }
+
+    return Napi::Boolean::New(env, out);
 }
 
 /**
- * Given an array of blobs and their proofs, verify that they corresponds to their
- * provided commitment.
+ * Given an array of blobs and their proofs, verify that they corresponds to
+ * their provided commitment.
  *
  * @remark blobs[0] relates to commitmentBytes[0] and proofBytes[0]
  *
  * @param[in] {Blob}    blobs - An array of serialized blobs to verify
- * @param[in] {Bytes48} commitmentBytes - An array of serialized commitments to verify
- * @param[in] {Bytes48} proofBytes - An array of serialized KZG proofs for verification
+ * @param[in] {Bytes48} commitmentBytes - An array of serialized commitments to
+ *                                        verify
+ * @param[in] {Bytes48} proofBytes - An array of serialized KZG proofs for
+ *                                   verification
```

Comment:
Not a fan of how these turned out.  They don't stack but I tried a couple options and this looks the least bad...  Open to suggestions @jtraglia

Reply:
Yeah, I don't think it looks great either but I also think it's the best option.